### PR TITLE
metadata translations may 2019

### PR DESCRIPTION
**Pull request by marqh**
translation updates from metarelate

### CODE REVIEW

File: iris_grib/_grib_cf_map.py [hunk 1]
```diff
@@ -218,20 +223,13 @@
     CFName('surface_downwelling_longwave_flux_in_air', None, 'W m-2'): G2Param(2, 0, 5, 3),
     CFName('surface_downwelling_shortwave_flux_in_air', None, 'W m-2'): G2Param(2, 0, 4, 7),
     CFName('surface_net_downward_longwave_flux', None, 'W m-2'): G2Param(2, 0, 5, 5),
-    CFName('surface_net_downward_longwave_flux', None, 'W m-2'): G2Param(2, 0, 5, 5),
```
Comment (bjlittle) [inline iris_grib/_grib_cf_map.py]: @marqh This removes a duplicate, thanks.

File: iris_grib/_grib_cf_map.py [hunk 2]
```diff
@@ -218,20 +223,13 @@
...
     CFName('surface_temperature', None, 'K'): G2Param(2, 0, 0, 17),
     CFName('surface_upward_latent_heat_flux', None, 'W m-2'): G2Param(2, 0, 0, 10),
     CFName('surface_upward_sensible_heat_flux', None, 'W m-2'): G2Param(2, 0, 0, 11),
     CFName('thickness_of_snowfall_amount', None, 'm'): G2Param(2, 0, 1, 11),
-    CFName(None, 'WAFC_CAT_potential', '1'): G2Param(2, 0, 19, 22),
-    CFName(None, 'WAFC_CB_horizontal_extent', '1'): G2Param(2, 0, 6, 25),
-    CFName(None, 'WAFC_ICAO_height_at_cloud_base', 'm'): G2Param(2, 0, 6, 26),
-    CFName(None, 'WAFC_ICAO_height_at_cloud_top', 'm'): G2Param(2, 0, 6, 27),
-    CFName(None, 'WAFC_icing_potential', '1'): G2Param(2, 0, 19, 20),
-    CFName(None, 'WAFC_in-cloud_turb_potential', '1'): G2Param(2, 0, 19, 21),
```
Comment (bjlittle) [inline iris_grib/_grib_cf_map.py]: @marqh These entries have just been moved, thanks.

File: iris_grib/_grib_cf_map.py [hunk 3]
```diff
@@ -116,8 +115,8 @@
     G2Param(2, 0, 6, 6): CFName('atmosphere_mass_content_of_cloud_liquid_water', None, 'kg m-2'),
     G2Param(2, 0, 6, 7): CFName('cloud_area_fraction_in_atmosphere_layer', None, '%'),
     G2Param(2, 0, 6, 25): CFName(None, 'WAFC_CB_horizontal_extent', '1'),
-    G2Param(2, 0, 6, 26): CFName(None, 'WAFC_ICAO_height_at_CB_base', 'm'),
-    G2Param(2, 0, 6, 27): CFName(None, 'WAFC_ICAO_height_at_CB_top', 'm'),
+    G2Param(2, 0, 6, 26): CFName(None, 'WAFC_ICAO_height_at_cloud_base', 'm'),
+    G2Param(2, 0, 6, 27): CFName(None, 'WAFC_ICAO_height_at_cloud_top', 'm'),
```
Comment (bjlittle) [inline iris_grib/_grib_cf_map.py]: @marqh This is a breaking change to the user, right?

Comment (marqh) [reply iris_grib/_grib_cf_map.py]: Hello @bjlittle 

this is an update, highlighting a mistake in the previous entries.

The term CB in WMO vocabularies is explicitly Cumulonimbus.
http://codes.wmo.int/ui/text-search?query=CB

These entries refer to 'convective cloud base'
http://codes.wmo.int/grib2/codeflag/4.2/0-6-26
which was interpreted by a domain expert as cloud base, but not Cumulonimbus.

This appears to be a semantic correction to me.
I don't infer from that that it is a 'breaking change'

It provides a consistency between the GRIB2 to CF Name and the CF Name to GRIB2 conversions which was previously missing.

Comment (bjlittle) [reply iris_grib/_grib_cf_map.py]: @marqh Thanks for the clarification :+1:

**Comment by bjlittle** [comment]
@marqh Great, thanks.

**Pull request merged**